.azure/test-linux.yml

```diff
@@ -46,6 +46,15 @@ jobs:
           path: .stestr
         displayName: "Cache stestr"
 
+      - ${{ if eq(parameters.testQPY, true) }}:
+        - task: Cache@2
+          inputs:
+            key: 'qpy | test/qpy_compat/test_qpy.py | "$(Build.BuildNumber)"'
+            restoreKeys: |
+              qpy | test/qpy_compat/test_qpy.py
+            path: qpy_files
+          displayName: cache old qpy
+
```

Comment:
Pretty nitpick-y, but perhaps we want to just fold this into the other `if testQPY` block?  You can have multiple elements of the array inside the condition, so we could keep things a little more localised in the file.

Reply:
I'll move it I don't feel strongly one way or the other. I only did this like this because my thinking was to do the cache loading all at once at the beginning of the job. But it functionally doesn't make a difference as long as this runs before the qpy test step.